services/QuillLMS/client/app/bundles/Diagnostic/test/actions/questions.test.ts

```diff
@@ -1,17 +1,17 @@
 import { mockFocusPointApi, } from '../__mocks__/focus_point_api'
 import { mockIncorrectSequenceApi, } from '../__mocks__/incorrect_sequence_api'
 import { mockQuestionApi, } from '../__mocks__/question_api'
+import { mockDispatch as dispatch, } from '../__mocks__/dispatch'
+import { SENTENCE_COMBINING_TYPE } from '../../libs/questions_api'
+import questionActions from '../../actions/questions'
+
 jest.mock('../../libs/questions_api', () => ({
   FocusPointApi: mockFocusPointApi,
   IncorrectSequenceApi: mockIncorrectSequenceApi,
   QuestionApi: mockQuestionApi,
 }))
 
-import { mockDispatch as dispatch, } from '../__mocks__/dispatch'
-
-import { SENTENCE_COMBINING_TYPE } from '../../libs/questions_api'
-
-import questionActions from '../../actions/questions'
+window.alert = jest.fn();
```

Comment:
this looks suspicious--but it seems you use `jest.fn()` later in the test. I'd move it closer to the call site to reduce confusion.

Reply:
My thinking was that this line is functioning as a file-wide mock, and we place these types of mocks directly below `import` statements.